### DISCUSSION TITLE
v0.8.2

### DISCUSSION THREAD

**Discussion by justinxzhao** [ludwig-ai/ludwig#3577] [General]
## What's Changed
* int: Rename original `combiner_registry` to `combiner_config_registry`, update decorator name by @ksbrar in https://github.com/ludwig-ai/ludwig/pull/3516
* Add mechanic to override default values for generation during model.predict() by @justinxzhao in https://github.com/ludwig-ai/ludwig/pull/3520
* [feat] Support for numeric date feature inputs by @jeffkinnison in https://github.com/ludwig-ai/ludwig/pull/3517
* Add new sythesized `response` column for text output features during postprocessing by @arnavgarg1 in https://github.com/ludwig-ai/ludwig/pull/3521
* Disable flaky twitter bots dataset loading test. by @justinxzhao in https://github.com/ludwig-ai/ludwig/pull/3439
* Add test that verifies that the generation config passed in at model.predict() is used correctly. by @justinxzhao in https://github.com/ludwig-ai/ludwig/pull/3523
* Move loss metric to same device as inputs by @Infernaught in https://github.com/ludwig-ai/ludwig/pull/3522
* Add comment about batch size tuning by @arnavgarg1 in https://github.com/ludwig-ai/ludwig/pull/3526
* Ensure user sets backend to local w/ quantization by @Infernaught in https://github.com/ludwig-ai/ludwig/pull/3524
* README: Update LLM fine-tuning config by @arnavgarg1 in https://github.com/ludwig-ai/ludwig/pull/3530
* Revert "Ensure user sets backend to local w/ quantization (#3524)" by @tgaddair in https://github.com/ludwig-ai/ludwig/pull/3531
* Revert "Ensure user sets backend to local w/ quantization" for release-0.8 branch and upgrade version to 0.8.1.post1 by @justinxzhao in https://github.com/ludwig-ai/ludwig/pull/3532
* Improve observability during LLM inference  by @arnavgarg1 in https://github.com/ludwig-ai/ludwig/pull/3536
* [bug] Pin pydantic to < 2.0 by @jeffkinnison in https://github.com/ludwig-ai/ludwig/pull/3537
* [bug] Support preprocessing `datetime.date` date features by @jeffkinnison in https://github.com/ludwig-ai/ludwig/pull/3534
* Remove obsolete prompt tuning example. by @justinxzhao in https://github.com/ludwig-ai/ludwig/pull/3540
* Add Ludwig 0.8 notebook to the README by @arnavgarg1 in https://github.com/ludwig-ai/ludwig/pull/3542
* Add `effective_batch_size` to auto-adjust gradient accumulation by @tgaddair in https://github.com/ludwig-ai/ludwig/pull/3533
* Refactor evaluation metrics to support decoded generated text metrics like BLEU and ROUGE. by @justinxzhao in https://github.com/ludwig-ai/ludwig/pull/3539
* Fix sequence generator test. by @justinxzhao in https://github.com/ludwig-ai/ludwig/pull/3546
* DRAFT: Revert "Add Cosine Annealing LR scheduler as a decay method (#3507)" by @justinxzhao in https://github.com/ludwig-ai/ludwig/pull/3545
* Set default max_sequence_length to None for LLM text input/output features by @arnavgarg1 in https://github.com/ludwig-ai/ludwig/pull/3547
* Add skip_all_evaluation as a mechanic to skip all evaluation. by @justinxzhao in https://github.com/ludwig-ai/ludwig/pull/3543
* Roll-forward with fixes: Fix interaction between scheduler.step() and gradient accumulation steps, refactor schedulers to use `LambdaLR`, and add cosine annealing LR scheduler as a decay method. by @justinxzhao in https://github.com/ludwig-ai/ludwig/pull/3555
* fix: Move model to the correct device for eval by @jeffkinnison in https://github.com/ludwig-ai/ludwig/pull/3554
* Report loss in tqdm to avoid log spam by @tgaddair in https://github.com/ludwig-ai/ludwig/pull/3559
* Wrap each metric update in try/except. by @justinxzhao in https://github.com/ludwig-ai/ludwig/pull/3562
* Move DDP model to device if it hasn't been wrapped yet by @tgaddair in https://github.com/ludwig-ai/ludwig/pull/3566
* ensure that there are enough colors to match the score index in visua… by @thelinuxkid in https://github.com/ludwig-ai/ludwig/pull/3560
* Pin Transformers to 4.31.0 by @arnavgarg1 in https://github.com/ludwig-ai/ludwig/pull/3569

## New Contributors
* @thelinuxkid made their first contribution in https://github.com/ludwig-ai/ludwig/pull/3560

**Full Changelog**: https://github.com/ludwig-ai/ludwig/compare/v0.8.1...v0.8.2

<hr /><em>This discussion was created from the release <a href='https://github.com/ludwig-ai/ludwig/releases/tag/v0.8.2'>v0.8.2</a>.</em>
